### DISCUSSION TITLE
Parameter name -> Mapping Name

### DISCUSSION THREAD

**Discussion by tompad2** [helgoboss/helgobox#1150] [Q&A] [answered]
Hi!

Very often when I does mappings for plugins I take the Parameter name and 
name Mapping name the same. Is there a way to do this automagic?

Or is this a possible feature request?

I have tried the Auto-name the listed mapping. but that didn't do it.

**Comment by helgoboss** [comment, marked as answer]
Auto-name should be improved, yes. Sounds like a feature request.
